docs/programming-oak.md

```diff
@@ -68,13 +69,10 @@ struct Node {
 <!-- prettier-ignore-end -->
 
 Under the covers the
-[`derive(OakExports)`](https://project-oak.github.io/oak/sdk/oak_derive/derive.OakExports.html)
-macro implements a main function named `oak_main` for you, with the following
-default behaviour:
-
-- Create an instance of your Node `struct` (using the `new()` method from the
-  [`Node`](https://project-oak.github.io/oak/sdk/oak/trait.Node.html) trait.
-- Take the channel handle passed to `oak_main()` and use it for gRPC input.
+[`entrypoint!`](https://project-oak.github.io/oak/sdk/oak/macro.entrypoint.html)
+macro implements a function identified by the name of the entrypount for you, 
+with the following default behaviour:
+- Take the channel handle passed to the entrypoint and use it for gRPC input.
```

Comment:
For total clarity maybe add another bullet:

 - Build an instance of the `Node` struct by using the constructor code included in the `entrypoint!` macro.

Reply:
Done.